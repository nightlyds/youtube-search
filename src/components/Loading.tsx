/* eslint-disable react/self-closing-comp */
import React from "react";

const Loading: React.FC = () => {
    return (
        <div className="loading">
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
            <div></div>
        </div>
    );
};

export default Loading;
