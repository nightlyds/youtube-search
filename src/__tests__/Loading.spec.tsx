import React from "react";
import Loading from "../components/Loading";
import { shallow } from "enzyme";

const setUp = () => shallow(<Loading />);

describe("Loading component tests", () => {
    let component: any;

    beforeEach(() => {
        component = setUp();
    });

    it("Snapshot test", () => {
        expect(component).toMatchSnapshot();
    });
});
