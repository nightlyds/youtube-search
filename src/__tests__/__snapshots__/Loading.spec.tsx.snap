// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component tests Snapshot test 1`] = `
<div
  className="loading"
>
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
  <div />
</div>
`;
